hack/quickstart-setup.sh

```diff
@@ -77,10 +89,19 @@ if [[ -z "${MGC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
   MGC_WORKLOAD_CLUSTERS_COUNT=1
 fi
 
+# TODO: remove before merge
+docker build . -t ${TOOLS_IMAGE} -f ./Dockerfile.tools
+
+# Make temporary directory for kubeconfig
+mkdir -p ${TMP_DIR}
+
 cleanupKind
 
 kindSetupMGCClusters ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_WORKLOAD} ${port80} ${port443} ${MGC_WORKLOAD_CLUSTERS_COUNT}
 
+# Export kubeconfig for control plane cluster
+kind export kubeconfig -n ${KIND_CLUSTER_CONTROL_PLANE} --kubeconfig ${TMP_DIR}/kubeconfig --internal
```

Comment:
This could be done in `kindCreateCluster` where the other kubeconfigs are generated, would work in local-setup and quick start then if we decided to just always use the docker tools image.